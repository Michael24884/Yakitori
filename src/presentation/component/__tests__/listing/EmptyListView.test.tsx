/**
 * @format
 */
import React from 'react';
import {EmptyListView} from '@components';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const instance = renderer.create(<EmptyListView />);
  expect(instance.toJSON()).toMatchSnapshot();
});
