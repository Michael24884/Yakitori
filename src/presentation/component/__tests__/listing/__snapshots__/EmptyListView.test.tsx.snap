// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      },
    ]
  }
>
  <Text
    style={
      Object {
        "color": "#FFFFFF",
      }
    }
    text="Oops!"
  >
    Oops!
  </Text>
  <Text
    style={
      Object {
        "color": "#4f4f4f",
        "fontSize": 8,
      }
    }
    text="Không có dữ liệu!"
  >
    Không có dữ liệu!
  </Text>
</View>
`;
